package.json

```diff
@@ -29,7 +29,7 @@
     "dev": "yarn workspace @blockprotocol/site dev",
     "dev:seed-db": "yarn workspace @blockprotocol/site dev:seed-db",
     "dev:db": "yarn workspace @blockprotocol/site dev:db",
-    "prepare": "patch-package && yarn workspace @blockprotocol/site patch-package && turbo run codegen && turbo run build \"--filter=!./site/\""
```

Comment:
This was required to patch `ajv` in `site`. It's no longer needed since the `ajv` patch has been moved to the root `patches` directory. It was moved because we also make use of ajv in `mock-block-dock` and have the apply the same patch there